snuba/cli/config.py

```diff
@@ -33,7 +33,7 @@ def config() -> None:
 
 
 @config.command("get-all")
-@click.option("--format", type=click.Choice(FORMATS.keys()), default="human")
+@click.option("--format", type=click.Choice([*FORMATS.keys()]), default="human")
```

Comment:
Is this something that is required by the new click version?

I see that click.Choice is used in other places which are not updated in this PR

Reply:
Yeah it is. Choice now expects a Sequence rather than an Iterable.